app.py

```diff
@@ -71,986 +100,1068 @@
+	elif "remind weekday ta hours" in text:
+		admin = list(db.child(key_fb_tars).child("admin").get().val())
+		if event_data["event"]["user"] not in admin:
+			tars.chat_postMessage(channel=event_data["event"]["channel"], text="You're not allowed to do this!")
+			return
+		tars.chat_postMessage(channel=sf_ta, text="Don't forget to mark your Mon-Thu TA Hours before 18:00, Sunday.")
+	elif "remind weekend ta hours" in text:
+		admin = list(db.child(key_fb_tars).child("admin").get().val())
+		if event_data["event"]["user"] not in admin:
+			tars.chat_postMessage(channel=event_data["event"]["channel"], text="You're not allowed to do this!")
+			return
+		tars.chat_postMessage(channel=sf_ta, text="Don't forget to mark your Fri-Sun TA Hours before 18:00, Thursday.")
+	elif "post weekday ta hours" in text:
+		admin = list(db.child(key_fb_tars).child("admin").get().val())
+		if event_data["event"]["user"] not in admin:
+			tars.chat_postMessage(channel=event_data["event"]["channel"], text="You're not allowed to do this!")
+			return
+		mon_thu_poll = db.child(key_fb_tars).child("tapoll").child("monthu").get().val()
+		poll = db.child(key_fb_tars).child("polls").child(mon_thu_poll).get().val()
+		db.child(key_fb_tars).child("polls").child(mon_thu_poll).remove()
+		text = "TA Hours for Monday to Thursday:\n"
+		for block in poll["message"][1:-3]:
+			try:
+				text += block["text"]["text"].split("`")[0].strip() + " " + block["text"]["text"].split("`")[2] + "\n"
+			except:
+				text += block["text"]["text"] + "\n"
+		tars.chat_delete(channel=sf_ta, ts=mon_thu_poll.replace("-", "."))
+		tars.chat_postMessage(channel=sf_ta, text=text)
+		tars.chat_postMessage(channel=orientation_id, text=text)
+		tars.chat_postMessage(channel=project_id, text=text)
+		tars.chat_postMessage(channel=general_id, text=text)
+	elif "post weekend ta hours" in text:
+		admin = list(db.child(key_fb_tars).child("admin").get().val())
+		if event_data["event"]["user"] not in admin:
+			tars.chat_postMessage(channel=event_data["event"]["channel"], text="You're not allowed to do this!")
+			return
+		fri_sun_poll = db.child(key_fb_tars).child("tapoll").child("frisun").get().val()
+		poll = db.child(key_fb_tars).child("polls").child(fri_sun_poll).get().val()
+		db.child(key_fb_tars).child("polls").child(fri_sun_poll).remove()
+		text = "TA Hours for Friday to Sunday:\n"
+		for block in poll["message"][1:-3]:
+			try:
+				text += block["text"]["text"].split("`")[0].strip() + " " + block["text"]["text"].split("`")[2] + "\n"
+			except:
+				text += block["text"]["text"] + "\n"
+		tars.chat_delete(channel=sf_ta, ts=fri_sun_poll.replace("-", "."))
+		tars.chat_postMessage(channel=sf_ta, text=text)
+		tars.chat_postMessage(channel=orientation_id, text=text)
+		tars.chat_postMessage(channel=project_id, text=text)
+		tars.chat_postMessage(channel=general_id, text=text)
+	elif "add publication" in text:
+		text_msg = "Visit https://sf-tars.herokuapp.com/addpublication to add the publication."
```

Comment:
```suggestion
		message = "Visit https://sf-tars.herokuapp.com/addpublication to add the publication."
```
Replace `text_msg` with `msg` for consistency.